editor/editor_file_system.cpp

```diff
@@ -1684,9 +1687,14 @@ void EditorFileSystem::_update_scene_groups() {
 			ProjectSettings::get_singleton()->add_scene_groups_cache(path, scene_groups);
 		}
 
-		ep.step(TTR("Updating Scene Groups..."), step_count++);
+		if (ep) {
+			ep->step(TTR("Updating Scene Groups..."), step_count++);
+		}
 	}
 
+	if (ep) {
+		memdelete(ep);
+	}
```

Comment:
For convenience you can use `memdelete_notnull` here instead

Reply:
Looks like it will be the first usage of this macro, unless I'm missing something 🤔

Reply:
Weird!